templates/AureliaSpa/ClientApp/app/components/navmenu/navmenu.html

```diff
@@ -0,0 +1,26 @@
+<template bindable="router">
+    <require from="./navmenu.css"></require>
+    <div class='main-nav'>
+        <div class='navbar navbar-inverse'>
+            <div class='navbar-header'>
+                <button type='button' class='navbar-toggle' data-toggle='collapse' data-target='.navbar-collapse'>
+                    <span class='sr-only'>Toggle navigation</span>
+                    <span class='icon-bar'></span>
+                    <span class='icon-bar'></span>
+                    <span class='icon-bar'></span>
+                </button>
+                <a class='navbar-brand' href="#/home">Aurelia</a>
+            </div>
+            <div class='clearfix'></div>
+            <div class='navbar-collapse collapse'>
+                <ul class='nav navbar-nav'>
+                    <li class="${row.isActive ? 'link-active' : ''}" repeat.for = "row of router.navigation" >
```

Comment:
For clarity, I would put the repeat.for first on the `li`, before the `class` binding.